.happy/config.json

```diff
@@ -2,14 +2,18 @@
     "config_version": "v1",
     "terraform_version": "0.13.5",
     "default_env": "rdev",
-    "app": "hosted-cellxgene",
-    "default_compose_env_file": ".env.ecr",
+    "app": "cellxgene",
```

Comment:
Not quite sure why we had a .happy/config.json file here since we're not using happy for this project?